hack/testdata/levee-config.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
This config is a cursory first-pass.  It will be iterated upon, and [KEP-1753](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1753-logs-sanitization/README.md) will introduce useful field tags to drive identification of `Source` fields that should not be logged.